src/validation.js

```diff
@@ -0,0 +1,26 @@
+const validation = (res, text, toCase) => {
```

Comment:
Function names should typically be verbs or verb phrases. Consider renaming the function from 'validation' to something like 'validateInput' to make its purpose clearer.